ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx

```diff
@@ -45,14 +48,16 @@ function getBaseNode(id: string): CustomNodeModel {
 function getNamespaceNode(
     namespace: string,
     cluster: string,
-    deploymentId: string
+    deploymentId: string,
+    isFiltered: boolean
 ): NamespaceNodeModel {
     const namespaceData: NamespaceData = {
         collapsible: true,
         showContextMenu: false,
         type: 'NAMESPACE',
         namespace,
         cluster,
+        labelIconClass: isFiltered ? filteredNamespaceSVG : derivedNamespaceSVG,
```

Comment:
That's a confusing property name that I guess the topology library dictates.

Reply:
Yes, it is very confusing as it's basically an icon, not a class 😕. Really wish they had some more in-depth docs for these sorts of things